install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Checking up on how we manage ConfigMaps, we currently [use `mergeMap` on `data`][1], and [`mergeMap`][2] allows other actors to append keys beyond those we list in the manifest.  That's unlikely to cause a problem, because actors with the auth to update this map are likely to also have the auth to ack any keys they inject here, but I thought I'd point it out in case others felt differently.

[1]: https://github.com/openshift/cluster-version-operator/blob/3b1047de4d2a82d2a1f4f92c224a1c46eba72ca9/lib/resourcemerge/core.go#L13-L19
[2]: https://github.com/openshift/cluster-version-operator/blob/3b1047de4d2a82d2a1f4f92c224a1c46eba72ca9/lib/resourcemerge/meta.go#L28-L41

Reply:
The code only checks the hardcoded keys so any additional data added is ignored.

Reply:
Verified through testing that he code only checks the hardcoded keys so any additional data added is ignored.